File: 2-AD-HOC/3454-Alice_no_Pais_do_Jogo_da_Velha-5/3454.py
x=str(input())
c=0
if x.count("O")!=1:
    print("?")
else:
    for i in range(0,2):
        if x[i]=="X" and x[i+1]=="X":
            print("Alice")
            c=1
    if x=="XOX":
        print("*")

File: 2-AD-HOC/3454-Alice_no_Pais_do_Jogo_da_Velha-5/3454.cs
using System; 

class URI {

    static void Main(string[] args) { 
        string x = Console.ReadLine();
        int c =0;
        foreach(char i in x){
            if (i=='O'){
                c++;
            }
        }
        if (c != 1){
            Console.WriteLine("?");
        }
        else{
            for(int j =0; j<2; j++){
                if(x[j] == 'X' & x[j+1] == 'X'){
                    Console.WriteLine("Alice");
                    c=1;
                }
            }
            if (x=="XOX"){
                Console.WriteLine("*");
            }
        }

    }

}